.github/workflows/cmake.yml

```diff
@@ -0,0 +1,53 @@
+name: CMake
+
+on: [push, pull_request]
+
+jobs:
+
+  build:
+    strategy:
+      matrix:
+        BUILD_TYPE: [Release, Debug]
+      
+    # The CMake configure and build commands are platform agnostic and should work equally
+    # well on Windows or Mac.  You can convert this to a matrix build if you need
+    # cross-platform coverage.
+    # See: https://docs.github.com/en/free-pro-team@latest/actions/learn-github-actions/managing-complex-workflows#using-a-build-matrix
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    
+    # install dependencies
+    - name: dependencies
+      run: sudo apt-get update && sudo apt-get install -yq openmpi-bin libfftw3-dev python3-dev libboost-all-dev python3-h5py
+
+    - name: Create Build Environment
+      # Some projects don't allow in-source building, so create a separate build directory
+      # We'll use this as our working directory for all subsequent commands
+      run: cmake -E make_directory ${{github.workspace}}/build
+
+    - name: Configure CMake
+      # Use a bash shell so we can use the same syntax for environment variable
+      # access regardless of the host operating system
+      shell: bash
+      working-directory: ${{github.workspace}}/build
+      # Note the current convention is to use the -S and -B options here to specify source 
+      # and build directories, but this is only available with CMake 3.13 and higher.  
+      # The CMake binaries on the Github Actions machines are (as of this writing) 3.12
+      run: cmake $GITHUB_WORKSPACE -DCMAKE_BUILD_TYPE=${{ matrix.BUILD_TYPE }} -DEXTERNAL_MPI4PY=OFF
+
+    - name: Build
+      working-directory: ${{github.workspace}}/build
+      shell: bash
+      # Execute the build.  You can specify a specific target with "--target <NAME>"
+      run: cmake --build . --target all -j 2
+
+    - name: Test
+      working-directory: ${{github.workspace}}/build
+      env:
+          PYTHONPATH: ${{github.workspace}}/build:${{github.workspace}}/build/contrib
```

Comment:
That should not be needed as CMake sets these paths automatically, see https://github.com/espressopp/espressopp/blob/05ae583ddabb47e959f867fc0b7539c6af2bd599/CMakeLists.txt#L176

Reply:
Ah, ok. But this is not compatible with `EXTERNAL_MPI4PY=OFF` as the `contrib` directory is missing. We could add the `contrib` directory here as well. However, I vote for removing the `TEST_ENV` entirely, because running the examples does not have the pre set-up environment. This way running tests and examples would need the same preparation steps.

Reply:
Huh, we had that in v2 https://github.com/espressopp/espressopp/blob/7a8e90426a53ecaf99225e682563542b6b1701cd/CMakeLists.txt#L190-L194

Reply:
Not sure why it got removed.

Reply:
What about removing it completely for consistency reasons?

Reply:
I would really like the test to work with having to set the `PYTHONPATH` externally, so we should add that back.

Reply:
I would prefer the other way, but well, not a big issue. I will add it.